src/components/structures/MatrixChat.js

```diff
@@ -424,40 +424,10 @@ module.exports = React.createClass({
                 }
                 break;
             case 'view_room_alias':
```

Comment:
view_room_alias now looks pretty similar to view_room. Is it worth merging them?